containers/account/DisableTwoFactorModal.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import {
+    Alert,
+    FormModal,
+    PasswordTotpInputs,
+    useLoading,
+    useApi,
+    useEventManager,
+    useNotifications
+} from 'react-components';
+import { disableTotp } from 'proton-shared/lib/api/settings';
+import { srpAuth } from 'proton-shared/lib/srp';
+import { PASSWORD_WRONG_ERROR } from 'proton-shared/lib/api/auth';
+
+const STEPS = {
+    CONFIRM: 1,
+    PASSWORD: 2
+};
+
+const DisableTwoFactorModal = (props) => {
+    const api = useApi();
+    const { call } = useEventManager();
+    const { createNotification } = useNotifications();
+    const [step, setStep] = useState(STEPS.CONFIRM);
+    const [password, setPassword] = useState('');
+    const [totp, setTotp] = useState('');
+    const [error, setError] = useState('');
+    const [loading, withLoading] = useLoading();
+
+    const { section, ...modalProps } = (() => {
+        if (step === STEPS.CONFIRM) {
+            return {
+                section: <Alert>{c('Message').t`Are you sure you want to disable two-factor authentication?`}</Alert>,
```

Comment:
What do you think about:
```suggestion
                section: <Alert type="warning">{c('Message').t`Are you sure you want to disable two-factor authentication?`}</Alert>,
```